src/pages/NotificationsPage/NotificationPage.tsx

```
@@ -1,192 +1,123 @@
+        <div className="flex items-start justify-center gap-2">
+          {NOTIFICATION_TABS.map((tab) => (
+            <button
+              key={tab.id}
+              onClick={() => setActiveTab(tab.id)}
+              className={cn(
+                "flex h-9 items-center justify-center gap-2 rounded-[12px] px-6 py-2 font-['Poppins'] text-[14px] leading-normal transition-colors",
+                activeTab === tab.id
+                  ? "bg-[#2563eb] font-medium text-white"
+                  : "border border-[#e4e4e4] bg-white font-normal text-[#6b7280] hover:bg-gray-50",
+              )}
             >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)